pkgs/development/python-modules/hypothesis/default.nix

```diff
@@ -9,28 +9,19 @@ buildPythonPackage rec {
   # pytz fake_factory django numpy pytest
   # If you need these, you can just add them to your environment.
 
-  version = "3.45.2";
+  version = "3.66.2";
   pname = "hypothesis";
 
-  # Upstream prefers github tarballs
-  src = fetchFromGitHub {
-    owner = "HypothesisWorks";
-    repo = "hypothesis-python";
-    rev = version;
-    sha256 = "063sn5m1966gvm3wrlxczdq4vw0r94h3nd9xpr94qxahpg2r4bpb";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0a4zfk9m9pdzqspassb3knfydn72ismxqb9ig7n4ryciiika8zd6";
   };
 
   checkInputs = [ pytest pytest_xdist flaky mock ];
   propagatedBuildInputs = [ attrs coverage ] ++ lib.optional (!isPy3k) [ enum34 ];
 
   inherit doCheck;
 
-  # https://github.com/DRMacIver/hypothesis/issues/300
-  checkPhase = ''
```

Comment:
This likely should be kept as well, unless they've changed their `tox.ini` to reduce the test scope.